tests/bfd/test_bfd.py

```diff
@@ -201,15 +241,57 @@ def create_bfd_sessions(ptfhost, duthost, local_addrs, neighbor_addrs, dut_init_
     if dut_init_first :
         ptfhost.shell(ptf_buffer)
 
+def create_bfd_sessions_multihop(ptfhost, duthost, loopback_addr, ptf_intf, nexthop_ip, neighbor_addrs):
```

Comment:
`nexthop_ip` is not used in this function.

Reply:
done